### PR TITLE
Revamp rules

### PR DESCRIPTION
**Pull request by residesJack**

### CODE REVIEW

File: src/pages/ServerRules.js [hunk 1]
```diff
@@ -23,46 +22,51 @@ class ServerRules extends Component {
 						<h2>General Composure</h2>
 						<ol>
 							<li>
-								Maintain a casual and positive atmosphere at all times. Be
-								respectful of all players. Harassment and bullying are strictly
-								not allowed.
+								Ensure that you are respectful to all players and are
+								maintaining a calm and positive atmosphere while on our server.
+								Any forms of harassment or bullying are strictly forbidden.
+								Don't be toxic.
```
Comment (BennyDoesTheStuff) [inline src/pages/ServerRules.js]: If we're going to include the text "Don't be toxic." we should be more specific on what we consider to be toxicity.

File: src/pages/ServerRules.js [hunk 2]
```diff
@@ -83,19 +87,18 @@ class ServerRules extends Component {
...
 							<li>
-								Map/stats abuse such as grinding alt accounts, spawn killing,
-								leaving the game right before getting killed, switching teams
-								right before you lose, and anything similar that may positively
-								affect your stats above other players on the server is not
-								allowed.
+								Grinding alt accounts, spawn killing, leaving the game right before getting
+								killed, switching teams right before you lose and anything similar
+								that may unfairly affect your stats is
+								not allowed. This is called map/stat abuse.
```
Comment (BennyDoesTheStuff) [inline src/pages/ServerRules.js]: I kinda preferred how this was phased before. Saying "This is called map/stat abuse." doesn't sound the best in my opinion.

File: src/pages/ServerRules.js (hunk 1 above)
Comment (residesJack) [reply src/pages/ServerRules.js]: edited

File: src/pages/ServerRules.js (hunk 2 above)
Comment (residesJack) [reply src/pages/ServerRules.js]: reverted back to its original state

**Pull request merged**